agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java

```diff
@@ -119,7 +119,7 @@ private BoolQueryBuilder getBaseQuery(List<String> entityIDs, String termID, boo
 
 		if (CollectionUtils.isNotEmpty(entityIDs)) {
 			for (String geneId : entityIDs) {
-				bool2.should(new MatchQueryBuilder("subject.curie.keyword", geneId));
+				bool2.should(new MatchQueryBuilder("subject.modEntityId.keyword", geneId));
```

Comment:
this "should" should have all three fields for the query. as in:
```
bool2.should(new MatchQueryBuilder("subject.curie.keyword", geneId));
bool2.should(new MatchQueryBuilder("subject.modEntityId.keyword", geneId));
bool2.should(new MatchQueryBuilder("subject.modInternalId.keyword", geneId));
```

Reply:
Makes sense.